.gitignore

```diff
@@ -36,4 +36,5 @@ Thumbs.db
 bower_components
 node_modules
 package-lock.json
+yarn.lock
```

Comment:
Maybe instead just use `npm` or delete/don't commit yarn's lock file.

Reply:
@fabiospampinato will do ... it has been removed